Dockerfile

```
@@ -10,25 +9,36 @@ ADD demo/inventory /runner/inventory
 ADD https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo /etc/yum.repos.d/ansible-runner.repo
 RUN dnf install -y epel-release && \
     dnf install -y ansible-runner python3-pip sudo rsync openssh-clients sshpass glibc-langpack-en git && \
+    alternatives --set python /usr/bin/python3 && \
     pip3 install ansible && \
-    chmod +x /bin/tini /bin/entrypoint && \
+    chmod +x /bin/tini && \
     rm -rf /var/cache/dnf
 
 RUN useradd runner && usermod -aG root runner
 
+ADD utils/entrypoint.sh /bin/entrypoint
+RUN chmod +x /bin/entrypoint
+
 # In OpenShift, container will run as a random uid number and gid 0. Make sure things
 # are writeable by the root group.
-RUN mkdir -p /runner/inventory /runner/project /runner/artifacts /runner/.ansible/tmp && \
-	chmod -R g+w /runner && chgrp -R root /runner && \
-	chmod g+w /etc/passwd
+RUN for dir in \
+      /runner \
```

Ran into some permissions errors. This fixes it:

```suggestion
      /runner \
      /home/runner \
```